### PR TITLE
Add description of behavior to explainer

### PR DESCRIPTION
**Pull request by jackbsteinberg**
Help expand on #29 and #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +159,18 @@ See [whatwg/html#4697](https://github.com/whatwg/html/issues/4697) for more disc
 
 ### The `<std-toast>` element
 
+#### Behavior
+
+The `<std-toast>` element provides a subtle, non-interruptive notification to the user.
+The toast will appear at a customized time and position on the screen, 
+and will typically contain a message and optionally action and dismiss buttons
+(though other markup in the element is supported).
```
Comment (fergald) [inline README.md]:
```suggestion
(though arbitrary markup in the element is supported).
```

Right?

Comment (jackbsteinberg) [reply README.md]: Yeah that's clearer

File: README.md [hunk 2]
```diff
@@ -159,6 +159,18 @@ See [whatwg/html#4697](https://github.com/whatwg/html/issues/4697) for more disc
...
+
+The `<std-toast>` element provides a subtle, non-interruptive notification to the user.
+The toast will appear at a customized time and position on the screen, 
+and will typically contain a message and optionally action and dismiss buttons
+(though arbitrary markup in the element is supported).
+The contents will be announced to a screen reader
+(politely or assertively depending on `type` [see [attributes](####attributes)]),
+and the optional action / dismiss buttons will appear next in the tabbing order.
+After a certain duration, the toast will timeout and hide itself,
+though this timeout will be suspended while the toast has focus or `mousehover`.
```
Comment (domenic) [inline README.md]:
```suggestion
though this timeout will be suspended while the toast has focus or the mouse is hovering over it.
```

(`mousehover` is not a defined term that I know of.)

File: README.md [hunk 3]
```diff
@@ -159,6 +159,18 @@ See [whatwg/html#4697](https://github.com/whatwg/html/issues/4697) for more disc
...
 ### The `<std-toast>` element
 
+#### Behavior
+
+The `<std-toast>` element provides a subtle, non-interruptive notification to the user.
+The toast will appear at a customized time and position on the screen, 
+and will typically contain a message and optionally action and dismiss buttons
+(though arbitrary markup in the element is supported).
+The contents will be announced to a screen reader
+(politely or assertively depending on `type` [see [attributes](####attributes)]),
```
Comment (domenic) [inline README.md]:
```suggestion
(politely or assertively depending on `type` [see [attributes](#attributes)]),
```

File: README.md [hunk 4]
```diff
@@ -159,6 +159,18 @@ See [whatwg/html#4697](https://github.com/whatwg/html/issues/4697) for more disc
...
+and the optional action / dismiss buttons will appear next in the tabbing order.
```
Comment (domenic) [inline README.md]:
```suggestion
and the focusable contents of the toast, if any, will appear next in the tabbing order.
```

**Review by domenic — APPROVED** [review]
LGTM with nits

File: README.md (hunk 2 above)
Comment (jackbsteinberg) [reply README.md]: Should I change this to `mouseover` instead? I think that's a defined term

Comment (domenic) [reply README.md]: Yeah, that's reasonable, I think. Depends on whether you want to use the event terms or just general English descriptions.

Comment (jackbsteinberg) [reply README.md]: I changed my mind, yours sounds better, going with that

**Pull request merged**